opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -190,7 +190,8 @@ impl UserEventsExporter {
                 // for each event.
                 // TODO: What if the event name is not provided? "Log" is used as default.
                 // TODO: Should event_tag be non-zero?
-                eb.reset(log_record.event_name().unwrap_or("Log"), 0);
+                let event_name = log_record.event_name().filter(|s| !s.is_empty()).unwrap_or("Log");
```

Comment:
What about whitespace only string?

Reply:
event_name() is used later in this file too, can you fix that too.